packages/atcommons/include/atcommons/at_expect.h

```diff
@@ -0,0 +1,6 @@
+#ifndef ATCOMMONS_EXPECT_H
+#define ATCOMMONS_EXPECT_H
+
+int atcommons_string_expect(char *actual, char *expected);
+
+#endif //ATCOMMONS_EXPECT_H
```

Comment:
This file should not be in the main include. This is a function which is defined and only used in the tests. This header file should be moved to the tests folder and it should be added as an include manually from the tests cmake file. This will cause an unresolved symbol when building with the tests disabled